release-notes/maxscale/script/generate_release_notes.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+
+if [ "$1" == "" ]
+then
+    echo "usage: $0 major.minor.patch [maturity]"
+    exit 1
+fi
+
+VERSION=$1
+
+patch=${VERSION##*.}
+major_minor=${VERSION%.*}
+major=${major_minor%%.*}
+minor=${major_minor##*.}
```

Comment:
A simpler approach would be to just require the caller to separate out the major, minor and patch components.
```
usage: $0 major minor patch [maturity]
```
This way there would be no need to parse any of it.

Reply:
Not really, it was just an idea for a simpler approach. I wasn't clear enough in my original message that this isn't an issue.